File: DoorsPOS.WinApp/frmManualInvGrid.vb
﻿Imports DevExpress.XtraGrid
Public Class frmManualInvGrid

    Private Sub frmManualInvGrid_FormClosed(sender As Object, e As FormClosedEventArgs) Handles Me.FormClosed
        Me.Dispose()
    End Sub

    Private Sub frmManualInvGrid_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        'TODO: This line of code loads data into the 'DsInitInv.Invntory' table. You can move, or remove it, as needed.
        Me.InvntoryTableAdapter.Fill(Me.DsInitInv.Invntory)
        'TODO: This line of code loads data into the 'DsManualInv.Invntory' table. You can move, or remove it, as needed.
    End Sub
    Private Sub GridControl1_DoubleClick(sender As Object, e As EventArgs) Handles GridControl1.DoubleClick
        ShowGridPreview(GridControl1)
    End Sub

    Sub ShowGridPreview(ByVal grid As GridControl)
        ' Check whether the GridControl can be previewed.
        If Not grid.IsPrintingAvailable Then
            MessageBox.Show("The Printing' library is not found", "Error")
            Return
        End If

        ' Opens the Preview window.
        grid.ShowPrintPreview()
    End Sub

End Class

File: DoorsPOS.WinApp/CS/frmManualInvGrid.cs
// VBConversions Note: VB project level imports
using System.Data;
using System.Collections.Generic;
using PDSA.Data;
using PDSA.WinForms.Forms;
using PDSA.Security;
using System.Diagnostics;
using PDSA.Common;
using PDSA.WinForms;
using PDSA.WinForms.Security;
using PDSA.PDSACryptography;
using DoorsPOS.Utility;
using Microsoft.VisualBasic;
using System.Collections;
using System;
using DoorsPOS.BusinessLayer;
using System.Windows.Forms;
using System.Drawing;
using System.Linq;
using DoorsPOS.DataLayer;
// End of VB project level imports

using DevExpress.XtraGrid;

namespace DoorsPOS.WinApp
{
	public partial class frmManualInvGrid
	{
		public frmManualInvGrid()
		{
			InitializeComponent();
		}
		
		public void frmManualInvGrid_FormClosed(object sender, FormClosedEventArgs e)
		{
			this.Dispose();
		}
		
		public void frmManualInvGrid_Load(object sender, EventArgs e)
		{
			//TODO: This line of code loads data into the 'DsInitInv.Invntory' table. You can move, or remove it, as needed.
			this.InvntoryTableAdapter.Fill(this.DsInitInv.Invntory);
			//TODO: This line of code loads data into the 'DsManualInv.Invntory' table. You can move, or remove it, as needed.
		}
		public void GridControl1_DoubleClick(object sender, EventArgs e)
		{
			ShowGridPreview(GridControl1);
		}
		
		public void ShowGridPreview(GridControl grid)
		{
			// Check whether the GridControl can be previewed.
			if (!grid.IsPrintingAvailable)
			{
				MessageBox.Show("The Printing\' library is not found", "Error");
				return ;
			}
			
			// Opens the Preview window.
			grid.ShowPrintPreview();
		}
		
	}
}
